packages/core-forger/src/defaults.ts

```diff
@@ -1,3 +1,8 @@
 export const defaults = {
-    hosts: [`http://127.0.0.1:${process.env.CORE_P2P_PORT || 4002}`],
+    hosts: [
```

Comment:
The `packages/core/bin/config/**/plugins.js` files also need to be updated. We can simply remove the defaults from the `plugins.js` files as the defaults in this file should rarely need to be changed.